frontend/src/components/HomePage/Profiles/profiles.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styles from './profiles.module.css';
+import pfpPlaceholder from '../../Assets/pfpPlaceholder.png';
+import locationIcon from '../../Assets/locationIcon.png'
+
+function UserProfile () {
```

Comment:
In the future, the component that makes the API request to search for user profiles will be the `<HomePage />` component, it will fetch a list of user profiles, and you'll need to render out each user profile using this `<UserProfile>` component. Knowing this, we should add some props to this component to dynamically render out each user profile based on, for example, their first name, last name, skill, bio and etc.

it would look like this:
```jsx
function UserProfile({ firstName, lastName, bio, skills }) {
   return (
       <h3> name: {firstName} {lastName} </h3>
   )
}
```
or you pass the entire user profile object to this UserProfile component
```jsx
function UserProfile(user) {
   return (
       <h3> name: {user.firstName} {user.lastName} </h3>
   )  
}

```